src/js/gi/gi-entry.jsx

```diff
@@ -0,0 +1,26 @@
+import 'core-js';
+require('../common');  // common javascript.
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Router, useRouterHistory } from 'react-router';
+import { Provider } from 'react-redux';
+import { createHistory } from 'history';
+import initReact from '../common/init-react';
+import routes from './routes';
+import { store } from './store';
```

Comment:
Use the common store `import { commonStore } from '../common/store';`

To allow it to work, you also need to go to `common/reducers/index.js` and import your GIBCT reducers. 
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/common/reducers/index.js

Then, in your GIBCT app, in the components that use state you can access them like so (depends on where you put GIBCT in the common store reducer):
 
This is an example from blue button

```
const mapStateToProps = (state) => {
  const bbState = state.health.bb;

  return {
    form: bbState.form,
  };
};
```